esmvaltool/preprocessor/_derive/clhmtisccp.py

```diff
@@ -0,0 +1,67 @@
+"""Derivation of variable `clhmtisccp`."""
+
+
+import iris
+from iris import Constraint
+
+from ._derived_variable_base import DerivedVariableBase
+
+
+class DerivedVariable(DerivedVariableBase):
+    """Derivation of variable `clhmtisccp`."""
+
+    def get_required(self, frequency):
+        """Get variable `short_name` and `field` pairs required for derivation.
+
+        Parameters
+        ----------
+        frequency : str
+            Frequency of the desired derived variable.
+
+        Returns
+        -------
+        list of tuples
+            List of tuples (`short_name`, `field`) of all variables required
+            for derivation.
+
+        """
+        return [('clisccp', 'T4' + frequency)]
+
+    def calculate(self, cubes, fx_files=None):
```

Comment:
I think it's very confusing that this function and many others have an fx_files input parameter that is never used. Can you come up with a more clear design?